examples/devices/zeromq_push_device.py

```diff
@@ -30,7 +30,7 @@ def __init__(
     def after_update(self):
         for address in self._addresses_to_publish:
             value = self.device.read(address)
-            self.send_message([{address: value}])
+            _ = self.send_message([{address: value}])
```

Comment:
Does pyright complain if you don't do this?

Reply:
>Result of async function call is not used; use "await" or assign result to variable

Reply:
Ah, should be

```suggestion
            await self.send_message([{address: value}])
```

Worrying that it didn't shout at me when I tested that...

Reply:
And make Adapter.after_update() async everywhere?

Reply:
Should this be something like `asyncio.create_task(send_message(...))` instead?

Reply:
I'm going to leave this as it is for now so that this PR is not a functional change. Will open a new PR for making the task actually get run